src/Microsoft.Azure.WebJobs.Host/Config/Apply.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Config
+{
+    // Used to register a converter that applies a New value to an existing value. 
+    // This is common for Streams. The new value is written to an existing stream instantation. 
+    class Apply<TNew, TExisting>
```

Comment:
Can you fix this type up? Apply isn't a good name for a top level type in this namespace. It's not clear what it represents, or in what context it should be used. Also, please add an explicit access modifier.